_includes/head.html

```diff
@@ -21,3 +21,8 @@
   {% asset styles.css %}
   {% asset animate.css %}
 </head>
+
+<body>
```

Comment:
Please move this to the default and minimal layouts. This file should have only `head` related content.